src/libraries/tests.proj

```diff
@@ -340,7 +350,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj"
+    <ProjectReference Include="$(MSBuildThisFileDirectory)*\tests\**\System.Buffers.Tests.csproj"
```

Comment:
Is this the case we only test `System.Buffers.Tests`? Can we go full BCL to see how long it takes on the devices?

Reply:
We need to get the size of each test suite down before it'll work.  We run out of space.

Reply:
As in disk space on the build machine?

Reply:
Yes, on the build machine.  We'll try to see if we can get it down and then enable everything.

Reply:
@steveisok @akoeplinger would it be possible to add the tvOS arm64 lane with a limited amount of BCL test suites so that we start collecting data on how much are the tvOS devices stable? Have maybe 10 test suites or so and leave it in runtime-staging?

We are currently starting to work on Apple/Android device telemetry so it would be nice to have some data to work with.